include/zephyr/sys/mem_manage.h

```diff
@@ -290,6 +290,13 @@ void z_phys_unmap(uint8_t *virt, size_t size);
  */
 size_t k_mem_free_get(void);
 
+/**
+ * Return the amount of free virtual memory available
+ *
+ * @return Free virtual RAM, in bytes
+ */
+int k_virt_free_range_max_get(void);
```

Comment:
Maybe `k_mem_virt_free_get()`? And this should return `size_t`.

The name used here somehow seems like it would return the range of memory.